Makefile

```diff
@@ -71,22 +71,22 @@ else
 endif
 
 # Build chaos-daemon binary
-chaosdaemon: generate fmt vet
+chaosdaemon: generate
```

Comment:
Can we add a single command `make check` for users to check code? 

```
check: fmt vet lint goimportxxxx
```

Reply:
🍻 I agree with you. Though adding `fmt` to `check` is a little wired, I cannot figure out a better name for it.